db/schema.rb

```diff
@@ -11,7 +11,16 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150317014350) do
+ActiveRecord::Schema.define(version: 20150413015135) do
+
+  create_table "forms", force: :cascade do |t|
```

Comment:
I think forms is a bad name for this.  This seems like data that is relevant to students.  Why not call it students?